src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -28,22 +30,76 @@ import {
   selectImageTypes,
   removePartition,
   selectPartitions,
+  changeIsNextButtonPristine,
+  selectIsNextButtonPristine,
+  selectFileSystemPartitionMode,
 } from '../../../../store/wizardSlice';
 import UsrSubDirectoriesDisabled from '../../UsrSubDirectoriesDisabled';
 import { ValidatedTextInput } from '../../ValidatedTextInput';
+import {
+  getDuplicateMountPoints,
+  isFileSystemConfigValid,
+} from '../../validators';
 
 export type Partition = {
   id: string;
   mountpoint: string;
   min_size: string;
 };
+type FileSystemConfigurationStepFooterPropTypes = {
+  setNextDisable(isNextDisable: boolean): void;
+  isNextDisable: boolean;
+};
+
+export const FileSystemStepFooter: React.FunctionComponent<
+  FileSystemConfigurationStepFooterPropTypes
+> = ({ setNextDisable, isNextDisable }) => {
```

Comment:
The prop typing should be applied to the props.

```suggestion
export const FileSystemStepFooter = ({
  setNextDisable,
  isNextDisable,
}: FileSystemConfigurationStepFooterPropTypes) => {
```